bin/plot_krun_results

```diff
@@ -698,24 +708,13 @@ class ProcessExecChart(object):
             self._plot_outliers(inset, large_markers=False)
             self._plot_changepoints(inset, large_markers=False)
             pyplot.draw()  # Force xticklabels to be drawn.
-            if self.x_bounds[0] > 0:
```

Comment:
OK, so it seems that if we give the script more sensible --xlimits then the xticklabels are integers, so I think we can leave that issue